### PR TITLE
[Docs] First swing at "genesis" release notes

### PR DESCRIPTION
**Pull request by jMyles**

### CODE REVIEW

File: docs/source/release_notes/genesis_release.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+8 And they called the re-encryption nodes Ursula. And the re-encryption and the decryption were the second secure message.
+9 And they said, Let the ciphertext fragments under the interwebs be gathered together unto one place, and let the Capsule appear: and it was so.
+10 And they called the total number of framents `n`, and the number that Bob needed to reassamble `m`: and they saw that it was good.
+11 And they said, Let the interwebs bring forth a TreasureMap, a Character, Alice, who yiels the Policy, and a Character, Enrico, who yield the encrypted message after his kind, whose symmetric key is in itself, upon the interwebs: and it was so.
+12 And the earth brought forth Alice, and Enrico, and Ursula, and Bob: and the decentralized dapp developer community saw that it was good.
+13 And the complete access management flow was the third secure message.
+14 And they said, Let there be a coherent metaphor and API in the  of cryptology of the interwebs, and let Policies expire at a specific datetime or upon revocation, and let them be for signs, and for seasons, and for days, and years:
```
Comment (mswilkison) [inline docs/source/release_notes/genesis_release.md]:
```suggestion
14 And they said, Let there be a coherent metaphor and API in the cryptology of the interwebs, and let Policies expire at a specific datetime or upon revocation, and let them be for signs, and for seasons, and for days, and years:
```

File: docs/source/release_notes/genesis_release.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+``# v0.1.0-alpha.0 Release Notes (Genesis Release)
+ 
+1 In the beginning cryptography created the ciphertext and the plaintext.
+2 And the ciphertext was without form, and void; and darkness was upon the face of the private key. And the spirit of assymetric crypto moved upon the face of the interwebs.
```
Comment (cygnusv) [inline docs/source/release_notes/genesis_release.md]:
```suggestion
2 And the ciphertext was without form, and void; and darkness was upon the face of the private key. And the spirit of asymmetric crypto moved upon the face of the interwebs.
```

File: docs/source/release_notes/genesis_release.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+5 And they called the symmetric ciphertext "bulk data", and they darkness the encapsulated key the "Capsule". And the encryption and the decryption were the first secure message.
+6 And they said, Let there be a re-encryption node in the midst of the interwebs, and let it divide the Bobs from the Bobs.
+7 And they made the re-encryption protocol, and divided the Bobs which were authorized under the Policy from the Bobs which were not authorized: and it was so.
+8 And they called the re-encryption nodes Ursula. And the re-encryption and the decryption were the second secure message.
+9 And they said, Let the ciphertext fragments under the interwebs be gathered together unto one place, and let the Capsule appear: and it was so.
+10 And they called the total number of framents `n`, and the number that Bob needed to reassamble `m`: and they saw that it was good.
+11 And they said, Let the interwebs bring forth a TreasureMap, a Character, Alice, who yiels the Policy, and a Character, Enrico, who yield the encrypted message after his kind, whose symmetric key is in itself, upon the interwebs: and it was so.
```
Comment (cygnusv) [inline docs/source/release_notes/genesis_release.md]:
```suggestion
11 And they said, Let the interwebs bring forth a TreasureMap, a Character, Alice, who yields the Policy, and a Character, Enrico, who yields the encrypted message after his kind, whose symmetric key is in itself, upon the interwebs: and it was so.
```

**Comment by jMyles** [comment]
I mean, to be clear: this was absolutely a joke PR.  I didn't intend this for merge.  

But if you all like it well enough, I'll fix it up and you can... I guess review it.

**Comment by michwill** [comment]
Lol. I think we certainly should merge, this is awesome

**Pull request merged**